src/py/flwr/client/app.py

```diff
@@ -141,6 +147,21 @@ def _parse_args_run_client_app() -> argparse.ArgumentParser:
         default="0.0.0.0:9092",
         help="Server address",
     )
+    parser.add_argument(
+        "--retry_max_tries",
+        type=int,
+        default=1,
+        help="The maximum number of times the client will try to reconnect to the"
+        "server before giving up in case of a connection error. If set to None,"
+        "there is no limit to the number of tries.",
```

Comment:
I think we may want to say that `--retry_max_tries` is the maximum number of times the client will try to connect (not reconnect) to the server. Because `--retry_max_tries 1` means that the client will work as before, trying to send the message once and then giving up. In this case, I think the client will not try to reconnect to the server. Wdyt?